src/transformers/tokenization_utils_base.py

```diff
@@ -2279,6 +2279,12 @@ def __call__(
         )
 
         if is_batched:
+            if isinstance(text_pair, str):
+                text_pair = [text_pair] * len(text)
+            if text_pair is not None:
+                assert len(text) == len(
+                    text_pair
+                ), f"batch length of `text`: {len(text)} does not match batch length of `text_pair`: {len(text_pair)}"
```

Comment:
With the formatting, this ends up harder to read than:
```suggestion
if text_pair is not None and len(text) != len(text_pair):
    raise ValueError(f"batch length of `text`: {len(text)} does not match batch length of `text_pair`: {len(text_pair)}.")
```